kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/flights/Flight.java

```diff
@@ -0,0 +1,52 @@
+package com.kodilla.good.patterns.flights;
+
+import java.util.Objects;
+
+public class Flight {
```

Comment:
lets make an assumption that Flight represents only one flight, without intermediate landing (currently you make an assumption that flight can have only one intermediate landing, but in reality there can be many of them).
In case that we are looking for flight via some city, you need to find 2 flighs:
from -> via
and via -> to